tt_metal/impl/dispatch/kernels/command_queue_consumer.hpp

```diff
@@ -116,7 +180,10 @@ FORCE_INLINE void write_buffers(
                 consumer_cb_size,
                 num_to_write,
                 page_size);
-            id += num_to_write;
+            page_id += num_to_write;
+        }
+        if (buffer_type == BufferType::SYSTEM_MEMORY) {
+            cq_push_back(command_issue_region_size, num_pages * page_size);
```

Comment:
I think it would be more readable with "completion_queue_push_back"

Reply:
makes sense. I will rename all the `cq_*` APIs on device and host to be `issue_queue_*` and `completion_queue_*`